Adding installation for Cloudflare Warp

julysfire opened the pull request:
Adding install instructions for Cloudflare Warp.  

Instructions include information about what Cloudflare Warp is and how it can help, install instructions for Windows including how to turn the service on after installation, Linux install instructions and pictures for instructions.

Also please ignore that I forgot to add a commit message, this is the first time I've used Gitbook.

julysfire committed:
No subject

julysfire committed:
Merge branch 'BBR-Troubleshooters:main' into main

Phyrok commented on connectivity/cloudflare-warp.md:
```diff
@@ -0,0 +1,170 @@
+
+# Cloudflare Warp
+
+Cloudflare Warp is a recursive DNS that tunnels your connection to the nearest Cloudflare data center where it can then be routed to the desired destination.  Instead of bouncing through a VPN or an ISP DNS service, Cloudflare will do the routing.  This can increase your connectivity to the game's servers.\
```
Remove \

Phyrok commented on connectivity/cloudflare-warp.md:
```diff
@@ -0,0 +1,170 @@
+
+1. Once complete, navigate to your downloads folder (C:\Users\\\<YourUsername>\Downloads).  You should see a file called "Cloudflare\_WARP\_Release-x64.msi".  Double click this to install.
+2. In the installer, click Next and then "Install".  When complete, click "Finish".
+3. On your System Tray in the bottom right, you should see a gray cloud icon.  If you don't, you might have to click the up arrow to show more icons.\
```
Remove \

Phyrok reviewed (changes requested):
Remove multiple " \ " from document.

What are the changes to the language.md?

Phyrok commented on .gitbook/assets/cloudflare3.png:
![image](https://github.com/BBR-Troubleshooters/Gitbook/assets/69722956/eb4924f4-fd3f-4fc6-89d6-9965b2e02561)

Try to make images the same size

Phyrok commented on .gitbook/assets/cloudflare4 (1).png:
duplicate

Phyrok commented on .gitbook/assets/cloudflare5.gif:
if possible remove black border on top & right

Phyrok commented on languages.md:
What changes have been made here?

Phyrok committed:
Update cloudflare-warp.md

Phyrok committed:
Update cloudflare-warp.md

Removed " \ " seems that code blocks are now closer together though

julysfire committed:
Updates for PR 5

Not sure the language.md changes -- Reverted.
Removed Duplicate image.
Resized image 3 and 4 to be the same.
Removed border from image 3 and 4.
Removed border from gif.

livingflore commented:
no need in it
